manifests/init.pp

```diff
@@ -34,13 +34,13 @@
 #                               type:Optional[Stdlib::Absolutepath]
 #
 # $forward::                    The forward option
-#                               type:Optional[String]
+#                               type:Optional[Enum['only', 'first']]
```

Comment:
Either's valid, in the same way you can use bare words in a manifest for single world strings. I'd prefer quoted.